tests/static_data/multi_publications/publications_test.py

```diff
@@ -16,5 +16,8 @@ def test_find_maps_containing_layer():
                          if publ_type == data.MAP_TYPE and (l_workspace, l_type, layer) in values[data.TEST_DATA].get('layers', [])}
 
         with app.app_context():
-            result_maps = pr_util.find_maps_containing_layer(l_workspace, layer)
+            result_maps = {
```

Comment:
Try to replace this static_data test method by dynamic_data test case with at lease one layer in two maps.